compiler/cpp/src/thrift/generate/t_swift_3_generator.cc

```diff
@@ -0,0 +1,2553 @@
+/*
```

Comment:
Assigning simple "--gen swift" to this one sounds good to me, given the Swift 2.x situation and this beeing pure Swift implementation.

In general, though, removing is hard.
The most natural course of action would be to keep current one as an generator switch like "swift:cocoa".

For completely replacement, we need ordinal considerations for any breaking changes.
How hard it is to migrate existing users' code ?
It seems to me that anyone who extended library code (inheritance etc) would be in a huge trouble.